src/main/rx/ghst.c

```diff
@@ -241,6 +241,7 @@ static bool ghstProcessFrame(const rxRuntimeState_t *rxRuntimeState)
             break;
 
             default:
+                return true;
                 break;
```

Comment:
The `break` and `return false` below are now not reachable. Probably a bit easier to maintain if you just change the `return false` below into a `return true`.